components/JoinUs.js

```diff
@@ -0,0 +1,79 @@
+import styled from 'styled-components'
+
+const JoinUsContainer = styled.div`
+  display: flex;
+  flex-direction: column;  
+  width: 750px;  
+  padding: 160px 0 158px 81px;
+`;
```

Comment:
Instead of creating a new container, can you use the `ContentContainer`? It's meant to be used internally in every section/page to uniformly style the content padding/width.

Reply:
Got it!